### PR TITLE
removes find_all_tags_truncate_on_abundance from python api

### PR DESCRIPTION
**Pull request by anotherthomas**
fixes issue #923

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, okay to test

**Comment by anotherthomas** [comment]
- [x] Is it mergeable?
- [x] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2015-04-13  Thomas Fenzl <thomas.fenzl@gmx.net>
+   * khmer/_khmermodule.cc: removed unused find_all_tags_truncate_on_abundance
```
Comment (luizirber) [inline ChangeLog]: Use a new line before listing the changes

**Comment by luizirber** [comment]
Fix the changelog formatting, and it is good to go!

**Comment by anotherthomas** [comment]
@luizirber ready for review and merge

**Comment by luizirber** [comment]
@anotherthomas Congratulations on your first commit to the khmer project! Your name will be included in the release notes for the next version and you'll be listed amongst our other contributors in the next software release paper.

**Pull request merged**